game-core/src/main/java/games/strategy/engine/chat/Chat.java

```diff
@@ -129,10 +128,6 @@ public void speakerRemoved(final INode node, final long version) {
             for (final IChatListener listener : listeners) {
               listener.addStatusMessage(node.getName() + " has left");
             }
-            playersThatLeftLast10.add(node);
-            if (playersThatLeftLast10.size() > 10) {
-              playersThatLeftLast10.remove(0);
-            }
```

Comment:
From a performance perspective, this is really a waste of memory.
Using a queue would've been so much more elegant (especially using something like guavas `EvictingQueue`).
Good that this code is removed.